navigator:
  arm_group: sia20d_arm
  gripper_group: end_effector
  wrist_link: link_t
  gripper_link: palm
  planner_service_name: /path_planning/plan_kinematic_path
  trajectory_filter_service_name: /trajectory_filter_server/filter_trajectory_with_constraints
  segmentation_service_name: /tabletop_segmentation
  grasp_planning_service_name: /plan_point_cluster_grasp
  planning_scene_service_name: /environment_server/set_planning_scene_diff
  arm_inverse_kinematics_plugin: SIA20D_Mesh_manipulator_kinematics/IKFastKinematicsPlugin
  joint_state_topic: /joint_states
segmentation:
  world_frame_id: base_link
  k_nearest: 80
  max_iterations: 400
  sac_method: 6 #ransac = 0
  normal_distance_weight: 0.2
  distance_threshold: 0.0005
  max_radius: 0.028
  min_fitness_score: 40 # minimum number of inliers
  align_to_top_centroid: true
goal:
  frame_id: base_link
  position:
    x: 0.451
    y: -0.493
    z: 0.2
  orientation:
    axis:
      x: 0
      y: 0
      z: 1
    angle: 0
  next_location:
    generation_mode: 0
    min_spacing: 0.05 # meters
    max_spacing: 0.08 # meters
    place_region_radius: 0.2 # meters
joints:
  side_position: [-0.4836713857380881, 0.6853144656754662, 2.4644586717834467, 1.2516597929426023, -0.41921308061550094, 1.3433802237698595, 1.1405494197356014]
